src/methods/get.js

```diff
@@ -312,7 +312,7 @@ export function GetPrototypeFromConstructor(
 }
 
 // ECMA262 7.3.1
-export function Get(realm: Realm, O: ObjectValue | AbstractObjectValue, P: PropertyKeyValue): Value {
+export function Get(realm: Realm, O: Value, P: PropertyKeyValue): Value {
```

Comment:
It allowed me to avoid writing the invariant in line 317 just once rather than in 6 different call sites. I'll restore it and write the 6 more invariants.